api_testrail.py

```diff
@@ -250,6 +257,67 @@ def testrail_milestones(self, project):
                 else:
                     print(f"No milestones data to insert into database for project {testrail_project_id}.") # noqa
 
+    def testrail_users(self):
+        # Step 1: Get all projects
+        projects_response = self.projects()
+        all_projects = projects_response.get("projects", [])
+        all_users = []  # List of all users across all projects
+        seen_project_ids = set()
+        project_user_counts = {}
+
+        for project in all_projects:
+            project_id = project["id"]
+            project_name = project["name"]
+
+            # Avoid hitting duplicate project IDs if that ever happens
```

Comment:
nit:  simplify comments to bare minimum for readability....
"# skip duplicate project IDs"